src/app/features/container/utils/get-popup-header.ts

```diff
@@ -39,20 +39,25 @@ export function showBalanceInfo(pathname: RouteUrls) {
   switch (pathname) {
     case RouteUrls.ProfileUpdateRequest:
     case RouteUrls.RpcSendTransfer:
-      return true;
+    case RouteUrls.PsbtRequest:
+    case RouteUrls.RpcSignBip322Message:
+    case RouteUrls.RpcStacksSignature:
     case RouteUrls.TransactionRequest:
+      return true;
     default:
       return false;
   }
 }
 
 export function getDisplayAddresssBalanceOf(pathname: RouteUrls) {
-  //  TODO it's unclear when to show ALL or STX balance here
   switch (pathname) {
-    case RouteUrls.TransactionRequest:
     case RouteUrls.ProfileUpdateRequest:
     case RouteUrls.RpcSendTransfer:
+    case RouteUrls.PsbtRequest:
+    case RouteUrls.RpcSignBip322Message:
       return 'all';
+    case RouteUrls.RpcStacksSignature:
+    case RouteUrls.TransactionRequest:
```

Comment:
This is feeding some code we have in this `TotalBalanceSuspense` which shows either a BTC balance or an STX balance:
```
function TotalBalanceSuspense({ displayAddresssBalanceOf }: TotalBalanceProps) {
  const account = useCurrentStacksAccount();
  const isBitcoinEnabled = useConfigBitcoinEnabled();
  return (
    <TotalBalanceLayout>
      <HStack alignItems="center" justifyContent="right">
        {account && displayAddresssBalanceOf === 'stx' && <StxBalance address={account.address} />}
        {isBitcoinEnabled && displayAddresssBalanceOf === 'all' && <BtcBalance />}
      </HStack>
    </TotalBalanceLayout>
  );
}
```

It's a bit confusing as there is a flag for `all` but in reality the code will only show either a BTC balance or and STX balance. 

I have set `RouteUrls.TransactionRequest` to use STX balance as when I check the code the route is wrapped by `StacksHighFeeWarningContainer` and the code seems very STX specific. If I don't do this then it will instead show the BTC balance

```
<Route
      path={RouteUrls.TransactionRequest}
      element={
        <AccountGate>
          <Suspense fallback={<SuspenseLoadingSpinner />}>
            <StacksHighFeeWarningContainer>
              <TransactionRequest />
            </StacksHighFeeWarningContainer>
          </Suspense>
        </AccountGate>
      }
    >
```